packager/transformer.js

```diff
@@ -17,15 +17,24 @@ const json5 = require('json5');
 const path = require('path');
 const ReactPackager = require('./react-packager');
 
-const babelRC =
-  json5.parse(
-    fs.readFileSync(
-      path.resolve(__dirname, 'react-packager', '.babelrc')));
 
-function transform(src, filename, options) {
-  options = options || {};
+const projectBabelRCPath = path.resolve(__dirname, '..', '..', '..', '.babelrc');
```

Comment:
This line only works for OS and if react native is in ``${projectDirectory}/node_modules/react-native``. I’m a little bit afraid of the apocalyptic potential of this innocent path lookup ;-)

Is anybody outside of Facebook using multiple project roots? The first project root might be a saner choice. Or we can add a command line option.